src/shared/lib/MutationUtils.ts

```diff
@@ -365,3 +365,10 @@ export function isValidGenomicLocation(mutation: Partial<Mutation>): boolean {
 
     return false;
 }
+
+export function hasASCNProperty(mutation: any, property: string) {
```

Comment:
Shouldn't be `mutation` a `Mutation`?
```suggestion
export function hasASCNProperty(mutation: Mutation, property: string) {
```